aptos/example/example_app/app/src/App.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import { PriceServiceConnection, PriceFeed } from "@pythnetwork/pyth-common-js";
+import { AptosClient } from "aptos";
+import { AptosPriceServiceConnection } from "@pythnetwork/pyth-aptos-js";
+
+const MAINNET_PRICE_SERVICE = "https://xc-mainnet.pyth.network";
+const TESTNET_PRICE_SERVICE = "https://xc-testnet.pyth.network";
+const APTOS_TESTNET_RPC = "https://testnet.aptoslabs.com/";
+
+const mainnetConnection = new PriceServiceConnection(MAINNET_PRICE_SERVICE); // Mainnet price service client to get visualize high frequency prices in the app
+const testnetConnection = new AptosPriceServiceConnection(
+  TESTNET_PRICE_SERVICE
+); // Price service client used to retrieve the offchain VAAs used to update the onchain price
+const aptosClient = new AptosClient(APTOS_TESTNET_RPC); // Aptos client is used to retrieve onchain prices
+
+// Account addresses
+const ETH_USD_MAINNET_PRICE_ID =
+  "0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace";
+const ETH_USD_TESTNET_PRICE_ID =
+  "0xca80ba6dc32e08d06f1aa886011eed1d77c77be9eb761cc10d72b7d0a2fd57a6";
+const PYTH_MODULE =
+  "0xaa706d631cde8c634fe1876b0c93e4dec69d0c6ccac30a734e9e257042e81541";
+const PYTH_TABLE_HANDLE =
+  "0x21b2122f77d3f9f944456c0ca8ffa6a13c541476433e64ab6ae81d48277a1181"; //The prices are stored in this table
+const MINT_NFT_MODULE = "_";
+
+function App() {
+  const [isConnected, setIsConnected] = React.useState<boolean>(false);
+
+  React.useEffect(() => {
+    window.aptos.disconnect();
+  }, []);
+
+  const [pythOffChainPrice, setPythOffChainPrice] = React.useState<number>(0);
+  const [pythOnChainPrice, setPythOnChainPrice] = React.useState<number>(0);
+
+  const fetchOnChainPrice = async () => {
+    let data = await aptosClient.getTableItem(PYTH_TABLE_HANDLE, {
```

Comment:
Instead of doing this can we make a call to `pyth::get_price`? Trying to discourage people from reading directly out of the table, as that's unsafe.

Reply:
It bypasses our recency checks, so consumers could be reading a stale price.

Reply:
I don't know how to call a function from client side.

Reply:
I don't think anyone should be trying to get the onchain price.
Frontends should be showing the offchain price.
In this example, it is just didactic to show the onchain price and the offchain price are two different concepts.
I think it's ok to do what I did as long as I put a warning comment.

Reply:
I think we should either:
- Not have this functionality at all
- Add a warning comment and a recency check (mimicking the on-chain recency check).